tests/e2e/run-fallback.sh

```diff
@@ -66,6 +66,17 @@ beforeEach
 
 beforeEach
 
+# test 'run' respects custom passphrase from environment
+DOPPLER_PASSPHRASE=123456 "$DOPPLER_BINARY" run -- echo -n > /dev/null
+# ensure default passphrase fails
+"$DOPPLER_BINARY" run --fallback-only -- echo -n > /dev/null 2>&1 && (echo "ERROR: --passphrase flag is not respected" && exit 1)
+# test decryption with custom passphrase flag
+"$DOPPLER_BINARY" run --fallback-only --passphrase=123456 -- echo -n > /dev/null || (echo "ERROR: --passphrase2 flag is not respected" && exit 1)
+# test decryption with custom passphrase from environment
+DOPPLER_PASSPHRASE=123456 "$DOPPLER_BINARY" run --fallback-only -- echo -n > /dev/null || (echo "ERROR: --passphrase3 flag is not respected" && exit 1)
```

Comment:
```suggestion
"$DOPPLER_BINARY" run --fallback-only --passphrase=123456 -- echo -n > /dev/null || (echo "ERROR: --passphrase flag is not respected" && exit 1)
# test decryption with custom passphrase from environment
DOPPLER_PASSPHRASE=123456 "$DOPPLER_BINARY" run --fallback-only -- echo -n > /dev/null || (echo "ERROR: --passphrase flag is not respected" && exit 1)
```

Reply:
@Piccirello I noticed this too when I was creating it. I assumed the incrementing number was to help you see which part of the test was failing easier. Was I not right there?

Reply:
I didn't realize this was an existing pattern. I believe this was added during local testing and never cleaned up. It does seem like it could help identify the failing line, though I'd probably put it at the end to avoid printing an incorrect flag name (i.e. `ERROR: --passphrase flag is not respected (2)`). Either way not a huge deal.